src/Evaluators/reduced_evaluator.jl

```diff
@@ -344,9 +360,7 @@ end
 function reset!(nlp::ReducedSpaceEvaluator)
     # Reset adjoint
     fill!(nlp.λ, 0)
-    # Reset initial state
-    copy!(nlp.x, initial(nlp.model, State()))
     # Reset buffer
-    nlp.buffer = get(nlp.model, PhysicalState())
+    init!(nlp.model, nlp.buffer)
```

Comment:
This is just a general comment. I like the mutliple dispatch in Julia, but I think it makes the function names at times a bit too generic. We have `get`, `init!`, `transfer`, etc, and the function signature has to tell what gets transferred to what, which can be ambiguous. `init_buffer` for example would be more readable to me.

Reply:
you are right about that. I will change the name of the function